labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -76,6 +133,19 @@ object CollectionExercise04 {
    * To keep it simple it's ok to use String.split to extract all words of a sentence.
    */
   def calcLengthLongestWord(lines: String*): Int = {
-    error("fix me")
+    if (lines.length == 0)
+      return 0
+    var longestWordList: List[Int] = lines.map(sizeOfLongestWord).toList
+    return maxOfList(longestWordList)
+  }
+  
+  def sizeOfLongestWord(sentence: String): Int = {
+    var wordList: List[String] = sentence.split(" ").toList
+    var wordSizeList: List[Int] = (wordList.map(_.length()))
+    return maxOfList(wordSizeList)
```

Comment:
Nice use of functional constructs!

Are these lists really `var` -iables though?